instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+class SearchBar extends React.Component {
+  render() {
+    return (
+      <input
+        type="text"
+        onChange={this.props.searchFilter}
+        name="search"
+        // value={this.props.newSearch}
+        placeholder="Search"
```

Comment:
Interesting what about the rest of the top bar including the instagram logo etc?